ably/proto/message_test.go

```diff
@@ -9,6 +12,40 @@ import (
 	. "github.com/ably/ably-go/Godeps/_workspace/src/github.com/onsi/gomega"
 )
 
+type CryptoData struct {
+	Algorithm string `json:"algorithm"`
+	Mode      string `json:"mode"`
+	KeyLen    int    `json:"keylength"`
+	Key       string `json:"key"`
+	IV        string `json:"iv"`
+	Items     []struct {
+		Encoded   proto.Message `json:"encoded"`
+		Encrypted proto.Message `json:"encrypted"`
```

Comment:
I'm curious to know why you choose to not use a named structure (i.e. `Item`). I suppose it's just a style choice.